docs/Theming.md

```diff
@@ -890,7 +890,7 @@ const App = () => (
 
 The `MenuItemLink` component make use of the React Router [NavLink](https://reacttraining.com/react-router/web/api/NavLink) component, hence allowing to customize its style when it targets the current page.
 
-If the default active style does not suit your tastes, you can override it by passing your own `classes`:
+If the default active style does not suit your tastes, you can override it by passing your own `classes`: // the code below doesn't use the `classes` prop
```

Comment:
The code below, introduced in 8fcd609e53ca1ff9ec1cf3635b7cdf2b5b42f5f3, doesn't teach anything about classes and doesn't show anything more than the previous snippet. It and should be removed, together with the entire sentence "if the default active style...".

Reply:
Done in https://github.com/marmelab/react-admin/pull/5546/commits/07e7ed26eeec8de6155988e04f43247d5b3d6feb